src/reducer/prodFilterReducer.js

```diff
@@ -0,0 +1,93 @@
+const HIGH_TO_LOW = "HIGH_TO_LOW";
+const LOW_TO_HIGH = "LOW_TO_HIGH";
+const PRICE_RANGE_FILTER = "PRICE_RANGE_FILTER";
+const CLEAR_FILTERS = "CLEAR_FILTERS";
+const CATEGORIES = "CATEGORIES";
+const GENDERSELECT = "GENDER";
+const BRANDS = "BRANDS";
+const rating = ["1", "2", "3", "4"];
+
+export function prodFilterReducer(state, action) {
+  const { payload } = action;
+  switch (action.type) {
+    case HIGH_TO_LOW:
+      return { ...state, sortBy: HIGH_TO_LOW };
+
+    case LOW_TO_HIGH:
+      return { ...state, sortBy: LOW_TO_HIGH };
+
+    case rating[3]:
```

Comment:
Instead of creating 4 different actions for each rating create single action as FILTER_BY_RATING and then change rating state from action payload(pass action payload with selected rating value while dispatching rating action)